ui/controls.js

```diff
@@ -353,6 +353,14 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     shaka.ui.ControlsPanel.elementNamesToFactories_.set(name, factory);
   }
 
+  /**
+   * @param {!shaka.extern.ISeekBarFactory} factory
+   * @export
+   */
+  static registerSeekBarFactory(factory) {
```

Comment:
Let's call this `registerSeekBar`, since the method next to it is `registerElement` and not `registerElementFactory`.

Reply:
Done ( even if registerFactory, makes more sens to me ). consistency, consistency : )